index.js

```diff
@@ -1 +1,38 @@
 // Code your solution in this file!
+function distanceFromHqInBlocks(num1){
+    if (num1 > 42) {
+      return num1 - 42;
+    } else {
+      return 42 - num1;
+    }  
+  };
+
+  function distanceFromHqInFeet(num) {
+    let feet = distanceFromHqInBlocks(num);
+      return feet * 264;
+  };
+  
+  function distanceTravelledInFeet(start, destination) {
+    let block;
+    if (start > destination) {
+      block = start - destination;
+    } else {
+      block = destination - start;
+    } return block * 264;
```

Comment:
Since you're not doing anything with the block, you don't need to assign it to a constant. 
```suggestion
    if (start > destination) {
      return start - destination;
    } else {
      return destination - start;
    }
```